README.md

```diff
@@ -134,6 +134,8 @@ The sequences (especially the last one) are pretty tough because of:
 * exposure time pretty high sometimes
 * camera images in the last sequence not sharp
 
+With "ssd_mobilenet_v1", the traffic light detections were correct in case of occurrence, but the availability was rather low.
```

Comment:
We never used ssd_mobilenet_v1, it was ssd_inception_v2_coco_2018_01_28. Please, fix.

Reply:
Oops, sorry. Fixed.